for_streamlit_only/data_customisation.py

```diff
@@ -55,10 +58,16 @@ def get_deg_win(chart_type, spectra_conversion_type, cols_right, df_columns):
     else:
         raise ValueError('Unknown chart type')
     
-    return vals
+    return df, vals
 
 
 def get_shifting_distance(spectra_conversion_type):
+    """
+    Shift spectra between each other.
```

Comment:
Wydaje mi się, że lepszym słowem na to co robi ten suwaj jest "separate"

Reply:
lepiej?